### PR TITLE
Added copy button for the sphinx page

### PR DESCRIPTION
**Pull request by kolibril13**
As seen here: https://sphinx-copybutton.readthedocs.io/en/latest/
Our examples in the docs will have small buttons on the top right to copy the code to clipboard.

### CODE REVIEW

**Comment by behackl** [comment]
Good idea!

If you are adding a new package, you'll need to add it to `docs/requirements.txt` as well.

File: docs/requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 Sphinx==3.1.2
 guzzle-sphinx-theme
 recommonmark>=0.5.0
+sphinx-copybutton
```
Comment (kolibril13) [inline docs/requirements.txt:4]: @behackl : just like this?

Comment (behackl) [reply docs/requirements.txt:4]: 👍

**Review by behackl — APPROVED** [review]
Looks great & works!

The little button is not very well visible, but I'll take care of that within #612.

**Comment by leotrs** [comment]
AnnotateBrace not working

Screenshot_20201024-173010

**Comment by behackl** [comment]
> AnnotateBrace not working

Not sure why, access via the link seems okay: https://manimce--611.org.readthedocs.build/en/611/_images/AnnotateBrace-1.png

**Comment by leotrs** [comment]
Huh.

(BTW, great PR)

**Comment by eulertour** [comment]
AnnotateBrace appears normally for me on PC and mobile.

**Comment by kolibril13** [comment]
Same for me.
@leotrs : This effect occured to me before at another image, I think is has something todo with how rtd is saving the data. Reloading the page after a few minutes solved the problwm.
And I think this only happens close after the build of the docs

**Comment by leotrs** [comment]
Confirmed, it's working for me now.

**Review by leotrs — APPROVED** [review]

**Pull request merged**